t/t2204-job-info.t

```diff
@@ -234,6 +234,21 @@ test_expect_success 'flux job list all jobs works' '
         test_cmp list_all_jobids.exp list_all_jobids.out
 '
 
+test_expect_success HAVE_JQ 'job stats lists jobs in correct state (mix)' '
+        depend=`flux job stats | jq .job_states.depend` &&
+        sched=`flux job stats | jq .job_states.sched` &&
+        run=`flux job stats | jq .job_states.run` &&
+        cleanup=`flux job stats | jq .job_states.cleanup` &&
+        inactive=`flux job stats | jq .job_states.inactive` &&
+        total=`flux job stats | jq .job_states.total` &&
```

Comment:
Just as an FYI, you can test JSON values directly in `jq`, e.g. `flux job stats | jq -e '.job_states.depend == 0`. No need to change anything here though.

Reply:
oh, haha, I knew that, but I was tweaking the old patch into the new one and just didn't think to tweak in that way.  I'll go ahead and tweak it to use `-e`.

Reply:
Ah, ok. I just wanted to make you aware if you weren't already since it is useful! :-)